lib/supplejack/story.rb

```diff
@@ -139,6 +139,17 @@ def self.find(id, user_key: nil, params: {})
       end
     end
 
+    # Fetches featured stories
+    # Oliver Stigley July 2017
+    #
+    def self.featured
+      begin
+        get('/stories/featured')
```

Comment:
@hapiben Do you agree?  I'm happy to take it off.  The view will still render with an empty array